data/packages/net.sam-tak.nudge.yml

```diff
@@ -1,8 +1,8 @@
 name: net.sam-tak.nudge
 displayName: Nudge
 description: Allows for adding comment and tasks to scene objects and project assets.
-repoUrl: 'https://github.com/SAM-tak/Nudge'
-parentRepoUrl: 'https://github.com/stevehalliwell/Nudge'
+repoUrl: 'https://github.com/SAM-tak/Nudge#upm'
+parentRepoUrl: 'https://github.com/stevehalliwell/Nudge#upm'
```

Comment:
The branch name is not part of the `repoUrl` or `parentRepoUrl`.

If you want OpenUPM to track your upm branch, simply create a new tag from the upm branch instead of the main branch.

Suppose you wish to keep both Git tags from the main branch and the upm branch. Please consider using a prefix. i.e. `1.1.0` for the main branch tag, `upm/1.1.0` for the upm branch tag. Then set the `gitTagPrefix` value of  `net.sam-tak.nudge.yml` to `upm/`.